ayaneo-platform.c

```diff
@@ -1041,12 +1500,30 @@ static int __init ayaneo_platform_init(void)
                 return -1;
         }
 
+        if(ps_priv.bypass_available) {
+            battery_hook_register(&battery_hook);
```

Comment:
Hans nixed using this in the upstream PR for oxp sensors and instead recommend power supply extension instead 

https://web.git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=6037802bbae892f3ad0c7b4c4faee39b967e32b0

Reply:
This is what I was talking about earlier, but realized that the kernel I was building against hadn't the feature available.
Will have a look, if I can build against this feature.